py-polars/tests/unit/functions/test_concat.py

```diff
@@ -23,31 +23,31 @@ def test_concat_lf_stack_overflow() -> None:
 
 
 def test_concat_horizontally_strict() -> None:
-    a = pl.DataFrame({"a": [0, 1, 2], "b": [1, 2, 3]})
-    b = pl.DataFrame({"c": [11], "d": [42]}) # 1 vs N (may broadcast)
-    c = pl.DataFrame({"c": [11, 12], "d": [42, 24]}) # 2 vs N
+    df1 = pl.DataFrame({"a": [0, 1, 2], "b": [1, 2, 3]})
+    df2 = pl.DataFrame({"c": [11], "d": [42]}) # 1 vs N (may broadcast)
+    df3 = pl.DataFrame({"c": [11, 12], "d": [42, 24]}) # 2 vs N
```

Comment:
Sorry--I meant here to align the `df_<X>` with the actual number of rows in the frame. So `pl.concat((df1, df3))` means "concatenate 1-row frame with 3-row frame".

Reply:
Oh.. Makes sense.
My bad